app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardFragment.java

```diff
@@ -208,7 +208,11 @@ public void showOrderStatistics(OrderStatistics orderStatistics) {
 
     @Override
     public void showError(String error) {
-        ViewUtils.showSnackbar(binding.getRoot(), error);
+        if (error.contains(getString(R.string.past_event_error_message))) {
+            ViewUtils.showSnackbar(binding.getRoot(), "Past events cannot be published.");
```

Comment:
@iamareebjamal "Past events cannot be published." Is this should be put inside strings too and do I need to also convert it into the Chinese language too.

Reply:
You NEVER, EVER should put something you match programmatically in strings.xml

Remove it and put this in strings.xml

Reply:
OK, Thanks